xAPI-Data.md

```diff
@@ -467,11 +468,12 @@ The table below lists all properties of an Identified Group.
 
 ###### Requirements
 
-* A system consuming Statements MUST consider each Anonymous Group distinct even if it has an identical set of members.
-* Activity Providers SHOULD use an Identified Group when they wish to issue multiple statements, aggregate data 
-or store and retrieve documents relating to a group.
-* An Activity Provider MAY include a complete or partial list of Agents in the 'member' property of a given Anonymous or Identified Group.
-* An LRS returning a statement MAY return the list of group members in any order.
+* A Learning Record Consumer MUST consider each Anonymous Group distinct even if it has an identical set of members.
+* Learning Record Providers SHOULD use an Identified Group when they wish to issue multiple statements, 
+aggregate data or store and retrieve documents relating to a group.
+* A Learning Record Provider MAY include a complete or partial list of Agents in the 'member' property of a 
```

Comment:
In fairness, I was terrible at following my own directions.  I'm going to leave that styling for the tech editor spin through this (if anyone sees these and PRs any before then, I won't hold it against you)